pkg/apis/hive/v1alpha1/validating-webhooks/syncset_validating_admission_hook.go

```diff
@@ -224,3 +245,43 @@ func checkValidPatchTypes(patches []hivev1.SyncObjectPatch) (string, bool) {
 	}
 	return "", true
 }
+
+func checkSecretReferences(secrets []hivev1.SecretReference) field.ErrorList {
+	allErrs := field.ErrorList{}
+	for i, secret := range secrets {
+		sourcePath := field.NewPath("spec").Child("secretReferences").Index(i).Child("source")
+		if len(secret.Source.Kind) == 0 {
+			allErrs = append(allErrs, field.Required(sourcePath.Child("kind"), "Kind is required"))
+		}
+		if secret.Source.Kind != "secret" {
```

Comment:
This function should validate the Group as well, which must be empty.

Reply:
Which group should be validated? The `ObjectReference` doesn't have group information, right?

Reply:
The group must be empty, which is the group for the core API.